core/src/main/java/org/apache/accumulo/core/clientImpl/InstanceOperationsImpl.java

```diff
@@ -282,19 +332,43 @@ public boolean testClassLoad(final String className, final String asTypeName)
   }
 
   @Override
+  @Deprecated
   public List<ActiveCompaction> getActiveCompactions(String server)
       throws AccumuloException, AccumuloSecurityException {
-    final var serverHostAndPort = HostAndPort.fromString(server);
 
+    HostAndPort hp = HostAndPort.fromString(server);
+
+    ServerId si = getServer(ServerTypeName.COMPACTOR, null, hp.getHost(), hp.getPort());
+    if (si == null) {
+      si = getServer(ServerTypeName.TABLET_SERVER, null, hp.getHost(), hp.getPort());
+    }
+    if (si == null) {
+      return new ArrayList<>();
+    }
+    return getActiveCompactions(si);
+  }
+
+  @Override
+  public List<ActiveCompaction> getActiveCompactions(ServerId server)
+      throws AccumuloException, AccumuloSecurityException {
+
+    @SuppressWarnings("unused")
+    var unused = Objects.nonNull(server);
```

Comment:
Again, wrong method. (The clue is needing to suppress warning :smiley_cat:)

```suggestion
    Objects.requireNonNull(server);
```

Reply:
Thanks, fixed in de7df88